joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/passes/AstCreator.scala

```diff
@@ -310,7 +309,7 @@ class AstCreator(filename: String, javaParserAst: CompilationUnit, global: Globa
     val parameterTypes = calcParameterTypes(method, typeParamValues)
 
     val returnType =
-      Try(method.getReturnType).toOption
+      Try(method.getReturnType)
         .map(returnType => typeInfoCalc.fullName(returnType, typeParamValues))
```

Comment:
Unless you have good reason to catch exceptions from the `map` please do not remove `toOption`.